doc/releases/quincy.rst

```diff
@@ -5,6 +5,1471 @@ Quincy
 Quincy is the 17th stable release of Ceph. It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+
+
+v17.2.8 Quincy
+==============
+
+This is the eighth, and expected to be last, backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+Users should expect to see the el8 rpm subdirectory empty and the "dnf" commands are expected
+to fail with 17.2.8.
```

Comment:
This doesn't say why or provide context.  It would be nice to mention that RPM packages for el9 are available instead.